### PR TITLE
RD2: Disable First Installment When Field Checked 

### PR DESCRIPTION
**Pull request by andrewyu-salesforce**
As an admin using integrations, I want to select which RDs have their first Opportunities created, on a per-record basis, so that I can handle different import scenarios.

# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata
- Custom Field: 
    - npe03__Recurring_Donation__c.DisableFirstInstallment__c

# Deleted Metadata

### CODE REVIEW

**Comment by andrewyu-salesforce** [comment]
@RadSFDeveloper @npsp-reedestockton The code is ready for review.

File: src/classes/RD2_OpportunityEvaluationService_TEST.cls [hunk 1]
```diff
@@ -1863,6 +1863,34 @@ private class RD2_OpportunityEvaluationService_TEST {
         
     }
 
+    /**
+    * @description Verifies that when Disable First Installment is true,
+    *   No installment will be created on RD insert regardless of NPSP Installment Auto Creation Setting
+    */
+    @IsTest
+    private static void shouldNotCreateFirstInstallmentWhenDisableFirstInstallmentIsTrue() {
```
Comment (RadSFDeveloper) [inline src/classes/RD2_OpportunityEvaluationService_TEST.cls:1871]: Can you please identify which existing tests map to AC1, AC2 and AC3? If they do not map, let's create a unit test for them.

Comment (andrewyu-salesforce) [reply src/classes/RD2_OpportunityEvaluationService_TEST.cls:1871]: [AC2](https://github.com/SalesforceFoundation/NPSP/blob/c24c3318768f8a1df3b1e714fba26f6bbba4a21c/src/classes/RD2_OpportunityEvaluationService_TEST.cls#L425), [AC3](https://github.com/SalesforceFoundation/NPSP/blob/c24c3318768f8a1df3b1e714fba26f6bbba4a21c/src/classes/RD2_OpportunityEvaluationService_TEST.cls#L343), and [AC4](https://github.com/SalesforceFoundation/NPSP/blob/c24c3318768f8a1df3b1e714fba26f6bbba4a21c/src/classes/RD2_OpportunityEvaluationService_TEST.cls#L367) is covered with NPSP Auto Creation Setting Tests.

**Comment by rponti-sforg** [comment]
QE testing is complete for this PR and associated WI.  Based on **WI Description**, **ACs**, and **Test Charter** feedback, tests have passed. 

For further WI and testing details see: 
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008UHNSIA4/view
https://salesforce.quip.com/WIj3AcFmAwa4

Proceeding with merging PR.  After PR is merged and builds (Beta Upload) have passed, then regression testing will begin against https://github.com/SalesforceFoundation/NPSP/pull/5748.  See **WI Feed** for further details regarding this regression effort.

**Pull request merged**